TimePicker/TimePickerBase.js

```diff
@@ -195,17 +195,36 @@ const TimePickerBase = kind({
 		 */
 		onMinuteChange: PropTypes.func,
 
+		/**
+		 * Called when the component is removed while retaining focus.
+		 *
+		 * @type {Function}
+		 * @param {Object} event
+		 * @public
+		 */
+		onSpotlightDisappear: PropTypes.func,
+
 		/**
 		 * Set content to RTL.
 		 *
 		 * @type {Boolean}
 		 * @private
 		 */
-		rtl: PropTypes.bool
+		rtl: PropTypes.bool,
+
+		/**
+		 * Disables 5-way spotlight from navigating into the component.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @public
+		 */
+		spotlightDisabled: PropTypes.bool
```

Comment:
Also, why do we need this?

Reply:
well, to disable 5-way navigation on the component. it is not clearly visible on Gallium, but it is on Silicon